config.json

```diff
@@ -171,6 +171,23 @@
 
       ]
     },
+    {
+      "uuid": "a9c2fbda-a1e4-42dd-842f-4de5bb361b91",
+      "slug": "dot-dsl",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 5,
+      "topics": [
+        "equality",
+        "classes",
+        "lists",
+        "domain_specific_languages",
+        "graphs",
+        "object_oriented_programming",
+        "test_driven_development",
+        "transforming"
+      ]
+    },
```

Comment:
The current version of exercism goes through the exercises in the order specified in the config. I don't think that this should be a particularly early exercise, so can you move this section down to around line 750 or so (the exact location isn't too important).

Reply:
Moved it lower!